_maps/_basemap.dm

```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
+#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```

Comment:
Remember to comment this out again.

Reply:
I will, there are still quite a few things I need to remove / atomize

Reply:
Another reminder to comment this back out.